test/org/javarosa/core/model/utils/test/DateUtilsTests.java

```diff
@@ -347,25 +279,21 @@ private LangJanSun(String language, String january, String sunday) {
             private String sunday;
         }
 
-        LangJanSun langJanSuns[] = new LangJanSun[] {
-            new LangJanSun("en", "Jan",   "Sun"),
-            new LangJanSun("es", "ene",   "dom"),
+        LangJanSun langJanSuns[] = new LangJanSun[]{
+            new LangJanSun("en", "Jan", "Sun"),
```

Comment:
I liked these columns better when they lined up. You undid my deliberate alignment, it seems.

Reply:
It's how IntelliJ reformats the code... sorry!

Reply:
But you didn’t restore the columns.

Reply:
I'm getting mixed signals that you want your columns back :)

I've restored them, but this is going to be a recurring thing after we set IntelliJ's code formatting rules to the project and everyone starts using automatic reformatting (which is highly needed on this and the other ODK projects). We can't go on arguing about whitespaces on merges... It's a waste of time!